### PR TITLE
{Release} Upgrade to Azure CLI 2.33.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,102 @@
...
+
+* `az acr connected-registry create`: Add `--notifications` to support adding patterns for generating notification events on connected registry artifacts (#20763)
+* `az acr connected-registry update`: Add `--add-notifications` and `--remove-notifications` to support adding or removing patterns for generating notification events on connected registry artifacts (#20763)
+
+**AKS**
+
+* `az aks nodepool add/update/upgrade`: Add new parameter `--aks-custom-headers` to support custom headers. (#21064)
+* `az aks create`: Add new parameter `--snapshot-id` to support creating a nodepool from snapshot when creating a cluster. (#21115)
+* `az aks nodepool add/upgrade`: Add new parameter  `--snapshot-id` to support creating a nodepool from snapshot. (#21115)
+* `az aks snapshot create/delete/list/show`: Add new commands to support the management of snapshot related operations. (#21115)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:19]:
```suggestion
* `az aks nodepool add/update/upgrade`: Add new parameter `--aks-custom-headers` to support custom headers (#21064)
* `az aks create`: Add new parameter `--snapshot-id` to support creating a nodepool from snapshot when creating a cluster (#21115)
* `az aks nodepool add/upgrade`: Add new parameter  `--snapshot-id` to support creating a nodepool from snapshot (#21115)
* `az aks snapshot create/delete/list/show`: Add new commands to support the management of snapshot related operations (#21115)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,102 @@
...
+
+**NetAppFiles**
+
+* `az netappfiles account ad add`: Added new optional parameter named encrypt-dc-connections (#20919)
+* `az netappfiles volume export-policy add`: Added missing optional parameters kerberos5_read_only, kerberos5_read_write, kerberos5i_read_only, kerberos5i_read_write, kerberos5_p_read_only, kerberos5_p_read_write, has_root_access, chown_mode (#20919)
+* `az netappfiles account ad update`: Command added (#21043)
+
+**Network**
+
+* Add microsoft.datafactory to supported private endpoints (#20895)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:76]:
```suggestion
* Add Microsoft.DataFactory/factories to supported private endpoints (#20895)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,102 @@
...
+* `az acr connected-registry update`: Add `--add-notifications` and `--remove-notifications` to support adding or removing patterns for generating notification events on connected registry artifacts (#20763)
+
+**AKS**
+
+* `az aks nodepool add/update/upgrade`: Add new parameter `--aks-custom-headers` to support custom headers. (#21064)
+* `az aks create`: Add new parameter `--snapshot-id` to support creating a nodepool from snapshot when creating a cluster. (#21115)
+* `az aks nodepool add/upgrade`: Add new parameter  `--snapshot-id` to support creating a nodepool from snapshot. (#21115)
+* `az aks snapshot create/delete/list/show`: Add new commands to support the management of snapshot related operations. (#21115)
+* Aks nodepool update --labels (#21095)
+* Aks update --nodepool-labels (#21095)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:21]:
```suggestion
* `az aks update/az aks nodepool update`: Allow empty string as label value
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,102 @@
...
+**NetAppFiles**
+
+* `az netappfiles account ad add`: Added new optional parameter named encrypt-dc-connections (#20919)
+* `az netappfiles volume export-policy add`: Added missing optional parameters kerberos5_read_only, kerberos5_read_write, kerberos5i_read_only, kerberos5i_read_write, kerberos5_p_read_only, kerberos5_p_read_write, has_root_access, chown_mode (#20919)
+* `az netappfiles account ad update`: Command added (#21043)
+
+**Network**
+
+* Add microsoft.datafactory to supported private endpoints (#20895)
+* Add microsoft.databricks to supported private endpoints (#20992)
```
Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:77]:
```suggestion
* Add Microsoft.Databricks/workspaces to supported private endpoints (#20992)
```

File: src/azure-cli/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,102 @@
...
+* `az image builder create`: Add new parameter `--proxy-vm-size` to support proxy VM size customization (#20904)
+* `az image builder create`: Add new parameter `--build-vm-identities` to  support user assigned identities customization (#20904)
+* `az vmss update`: Add new parameter `--force-deletion` to support force delete VMSS (#20622)
+* `az vm/vmss create`: Add warning log and modify help to inform that the default value `Contributor` of `--role` will be removed (#20924)
+* `az disk-encryption-set create`: Make the parameter `--source-vault` un-required (#20256)
+* `az vm create/update`: Add new parameters `--v-cpus-available`  and `--v-cpus-per-core` to support  VMSize customization (#20818)
+
+**Cosmos DB**
+
+* `az managed-cassandra cluster status`: Add table format support to /status api (#21131)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:58]:
```suggestion
* `az managed-cassandra cluster status`: Add table format support (#21131)
```

File: src/azure-cli/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,102 @@
...
+**App Config**
+
+* [BREAKING CHANGE] Support app service slots (#20850)
+
+**App Service**
+
+* `az webapp vnet-integration add `: Fix a bug that prevented adding a vnet in a different subscription from the webapp (#20910)
+* `az functionapp vnet-integration add `: Fix a bug that prevented adding a vnet in a different subscription from the functionapp (#20910)
+* `az webapp create `: Support joining a vnet in a different subscription (#20910)
+* `az functionapp create `: Support joining a vnet in a different subscription (#20910)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:32]:
```suggestion
* `az webapp create`: Support joining a vnet in a different subscription (#20910)
* `az functionapp create`: Support joining a vnet in a different subscription (#20910)
```

File: src/azure-cli/HISTORY.rst [hunk 7]
```diff
@@ -3,6 +3,102 @@
...
+* Aks update --nodepool-labels (#21095)
+
+**App Config**
+
+* [BREAKING CHANGE] Support app service slots (#20850)
+
+**App Service**
+
+* `az webapp vnet-integration add `: Fix a bug that prevented adding a vnet in a different subscription from the webapp (#20910)
+* `az functionapp vnet-integration add `: Fix a bug that prevented adding a vnet in a different subscription from the functionapp (#20910)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:30]:
```suggestion
* `az webapp vnet-integration add`: Fix a bug that prevented adding a vnet in a different subscription from the webapp (#20910)
* `az functionapp vnet-integration add`: Fix a bug that prevented adding a vnet in a different subscription from the functionapp (#20910)
```

File: src/azure-cli/HISTORY.rst [hunk 8]
```diff
@@ -3,6 +3,102 @@
...
+* `az keyvault create`: Add default permissions on keyvault creation (#20937)
+
+**Monitor**
+
+* `az monitor action-group`: Support event hub receiver (#21059)
+
+**NetAppFiles**
+
+* `az netappfiles account ad add`: Added new optional parameter named encrypt-dc-connections (#20919)
+* `az netappfiles volume export-policy add`: Added missing optional parameters kerberos5_read_only, kerberos5_read_write, kerberos5i_read_only, kerberos5i_read_write, kerberos5_p_read_only, kerberos5_p_read_write, has_root_access, chown_mode (#20919)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst]:
```suggestion
* `az netappfiles account ad add`: Add new optional parameter named encrypt-dc-connections (#20919)
* `az netappfiles volume export-policy add`: Add missing optional parameters kerberos5_read_only, kerberos5_read_write, kerberos5i_read_only, kerberos5i_read_write, kerberos5_p_read_only, kerberos5_p_read_write, has_root_access, chown_mode (#20919)
```

File: src/azure-cli/HISTORY.rst [hunk 9]
```diff
@@ -3,6 +3,101 @@
...
+* `az batch create/activate`: Add clarify application package path help info for argument `--package-file` (#21000)
+
+**Bot Service**
+
+* `az bot create`: Add location as specified by user to bot creation for regionality/EUDB (#20716)
+
+**Compute **
+
+* `az image builder create`: Add new parameter `--proxy-vm-size` to support proxy VM size customization (#20904)
+* `az image builder create`: Add new parameter `--build-vm-identities` to  support user assigned identities customization (#20904)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:49]:
```suggestion
* `az image builder create`: Add new parameter `--build-vm-identities` to support user assigned identities customization (#20904)
```

File: src/azure-cli/HISTORY.rst [hunk 10]
```diff
@@ -3,6 +3,101 @@
...
+* `az vmss update`: Add new parameter `--force-deletion` to support force delete VMSS (#20622)
+* `az vm/vmss create`: Add warning log and modify help to inform that the default value `Contributor` of `--role` will be removed (#20924)
+* `az disk-encryption-set create`: Make the parameter `--source-vault` un-required (#20256)
+* `az vm create/update`: Add new parameters `--v-cpus-available`  and `--v-cpus-per-core` to support  VMSize customization (#20818)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:53]:
```suggestion
* `az vm create/update`: Add new parameters `--v-cpus-available` and `--v-cpus-per-core` to support VMSize customization (#20818)
```

**Review by calvinhzy — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Comment by yonzhan** [comment]
Azure CLI 2.33.0 release

**Pull request merged**